runtime/src/main/scala/akka/grpc/internal/ByteStringUtils.scala

```diff
@@ -0,0 +1,44 @@
+package akka.grpc.internal
+
+import akka.annotation.InternalApi
+import akka.util.ByteString
+import io.grpc.KnownLength
+
+import java.io.{ ByteArrayOutputStream, InputStream }
+
+@InternalApi
+private[grpc] object ByteStringUtils {
+  def fromInputStream(stream: InputStream): ByteString = {
+    val buffer =
```

Comment:
I moved the old code to this Utils object so we can provide a default implementation in ProtobufSerializer for compat reasons.